site/content/contribute/more-info/webapp/e2e-testing.md

```diff
@@ -13,9 +13,9 @@ aliases:
   - contribute/more-info/webapp/e2e/troubleshooting
 ---
 
-End-to-end tests for the Mattermost webapp in general use {{<newtabref href="https://www.cypress.io/" title="Cypress">}} and {{<newtabref href="https://playwright.dev/" title="Playwright">}}. If you're not familiar with Cypress, check out the Cypress {{<newtabref href="https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell" title="Developer Guide">}} and {{<newtabref href="https://docs.cypress.io/api/api/table-of-contents.html" title="API Reference">}}. Feel free to also join us on the Mattermost Community server if you'd like to ask questions and collaborate with us!
+End-to-end tests for the Mattermost web app in general use {{<newtabref href="https://www.cypress.io/" title="Cypress">}} and {{<newtabref href="https://playwright.dev/" title="Playwright">}}. If you're not familiar with Cypress, check out the Cypress {{<newtabref href="https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell" title="Developer Guide">}} and {{<newtabref href="https://docs.cypress.io/api/api/table-of-contents.html" title="API Reference">}}. Feel free to also join us on the Mattermost Community server if you'd like to ask questions and collaborate with us!
```

Comment:
I didn't make too many changes to this document outside of updating paths and links, so it could probably use another pass by QA to double check that I didn't miss anything